apps/watch/src/libs/videoContentFields.ts

```diff
@@ -19,6 +19,17 @@ export const VIDEO_CONTENT_FIELDS = gql`
     studyQuestions(languageId: $languageId, primary: true) {
       value
     }
+    bibleCitations {
```

Comment:
Does bible citations have translations? Can you add this if it has it?
```suggestion
    bibleCitations(languageId: $languageId, primary: true) {
```

Reply:
They do not